src/WebJobs.Script/Rpc/LanguageWorkerService.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Script.Abstractions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    internal class LanguageWorkerService : ILanguageWorkerService, IConfigureOptions<ScriptJobHostOptions>
+    {
+        private IEnumerable<WorkerConfig> _languageWorkerConfigs;
+
+        internal LanguageWorkerService(IConfiguration rootConfig)
+        {
+            ILoggerFactory loggerFactory = NullLoggerFactory.Instance;
```

Comment:
@brettsam / @fabiocav  - Please let me know if there is a way to get startUp logger here

Reply:
I didn't see where you were adding this to the builder (see my comment above). If you did, then you'd get an ILoggerFactory from DI

Reply:
Still figuring this out. Logger is still null.